### PR TITLE
ANDROID-14548 update close translation

### PR DESCRIPTION
**Pull request by jeprubio**
### :goal_net: What's the goal?
Update string used for closing the bottom sheet as content description.
### :construction: How do we do it?
- Asking for the translation and updating it.
The translations:
es_es: Cerrar ventana
de_de: Fenster schließen
en_us: Close window
pt_br: Fechar janela

### ☑️ Checks
- [x] I updated the documentation, including readmes and wikis. If this is a breaking change, update [UPGRADING.md](../UPGRADING.md) to inform users how to proceed. If no updates are necessary, indicate so.
- [ ] Tested with dark mode.
- [ ] Tested with API 21.

### :test_tube: How can I test this?
- [x] 🖼️ Screenshots/Videos

(turn the volume ON)

https://github.com/Telefonica/mistica-android/assets/13270085/6adbb667-10e5-4f6f-9264-5e888b3659d4

- [ ] Mistica App QR or download link

### CODE REVIEW

File: library/src/main/res/values-pt/strings_content_descriptions.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="close_button_content_description" comment="Content description for image views that act as buttons for dismissing views">
-Fechar</string>
+Fechar janela</string>
```
Comment (yamal-alm) [inline library/src/main/res/values-pt/strings_content_descriptions.xml:4]: I'm not sure if this token with an initial line break is being treated as one single line or two 🤔 just thinking aloud, maybe we should align this

**Review by yamal-alm — APPROVED** [review]
nice 👍

File: library/src/main/res/values-pt/strings_content_descriptions.xml (hunk 1 above)
Comment (jeprubio) [reply library/src/main/res/values-pt/strings_content_descriptions.xml:4]: Ups, it's true there is a line break, I'll remove it just in case.

**Review by pmartinbTEF — APPROVED** [review]
LGTM

**Pull request merged**